src/eterna/pose2D/PoseField.ts

```diff
@@ -103,31 +107,94 @@ export default class PoseField extends ContainerObject implements KeyboardListen
     }
 
     private onMouseDown(e: InteractionEvent): void {
-        if (!Flashbang.app.isControlKeyDown) {
-            this.cancelDrag();
+        if (!isMobile()) {
```

Comment:
I'm not entirely sure what the purpose of this guard is, so please disregard this if I've just misunderstood something. Should the check be for if it's on mobile or not, or should it be if the device has a touchscreen or not? I'm thinking something like `e.data.pointerType === 'touch'`.

Reply:
@Zguy I agree, `e.data.pointerType === 'touch` is much cleaner for this purpose which is to disregard this event if using a mouse and not using the left button. Will change it!

Reply:
There is already a flashbang helper method (in InputUtil I think?) to check if an event is the left mouse button/primary pointer device - maybe create another such method for this?